analyticConfigurations/computed-aws.nlb.json

```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "netuitive.aws.networkelb.unhealthyhostpercent",
+        "properties": {
+          "expression": "(data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual) == 0 ? 0 : (data['aws.networkelb.unhealthyhostcount'].actual / (data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual))*100",
+          "fqn": "netuitive.aws.networkelb.unhealthyhostpercent",
```

Comment:
let's not prefix this with 'netuitive' just remove it, we've been slowly moving away from this approach and even calculating these locally where possible.  So the fqn would just be aws.networkelb.unhealthyhostpercent